### PR TITLE
adding the node_modules ignore instructions to the readme

### PR DESCRIPTION
**Pull request by tylercherpak**
### Checklist:

- [ ] I have read the [**CONTRIBUTING**](/CONTRIBUTING.md) document.
- [ ] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have added tests to cover my change.
- [ ] All new and existing tests passed.

### CODE REVIEW

**Comment by tylercherpak** [comment]
hey @christianc1 can you review this readme change about the node_modules issue? Thanks! 😄

File: README.md [hunk 1]
```diff
@@ -267,6 +267,27 @@ somewhere within `~/wp-local-docker-sites/<environment>/`).
...
+    volumes:
+        - './wordpress:/var/www/html:cahced`
+        - '/var/www/html/wp-content/themes/{my-theme}/node_modules`
+        - '/var/www/html/wp-content/plugins/{my-plugin}/node_modules`
+phpfpm:
+    ...
+    volumes:
+        - './wordpres:/var/www/html:cached'
+        - '/var/www/html/wp-content/themes/{my-theme}/node_modules`
+        - '/var/www/html/wp-content/plugins/{my-plugin}/node_modules`
```
Comment (tomalec) [inline README.md:286]: There is an inconsistency in string delimiters:

```suggestion
        - './wordpress:/var/www/html:cahced'
        - '/var/www/html/wp-content/themes/{my-theme}/node_modules'
        - '/var/www/html/wp-content/plugins/{my-plugin}/node_modules'
phpfpm:
    ...
    volumes:
        - './wordpres:/var/www/html:cached'
        - '/var/www/html/wp-content/themes/{my-theme}/node_modules'
        - '/var/www/html/wp-content/plugins/{my-plugin}/node_modules'
```

Comment (tylercherpak) [reply README.md]: Thanks for catching that!! 😄 Accepted!

**Review by jeffpaul — APPROVED** [review]
I made some small tweaks, but othewise this reads well to me.  Unless you're concerned about the sample snippet for the YML file, then I'd say good to merge.

**Review by jeffpaul — APPROVED** [review]
majestic :chefs-kiss:

**Pull request merged**